ProjectMars/Features/Skill.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Skill Feature
+Scenario: A.Add a Skill to User Profile
+Given User Login to MarsPortal Application Successfully
+When Create a New Skill into User Profile '<Skill>' and '<Level>'
+Then The Skill Record Created '<Skill>' and '<Level>'
```

Comment:
Please increase your coverage around the features you  are testing here is an idea of what is missing 

• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing